nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get('/calculator/multiply',  (req, res) => {
+    const values = Object.values(req.query).flat().reduce((acc, val) => (acc) * (val))
+    res.send(` ${values}`)
+});
+app.get('/calculator/add',  (req, res) => {
+    const values = Object.values(req.query).flat().reduce((acc, val) => ((acc)*1 + (val)*1))
+    res.send(` ${values}`)
+});
+app.get('/calculator/substract',  (req, res) => {
```

Comment:
```suggestion
app.get('/calculator/subtract',  (req, res) => {
```

Reply:
Thanks, for your suggestion. If query parameters are not numbers then I get NaN value. 
Yes, I used Postman for post requests.